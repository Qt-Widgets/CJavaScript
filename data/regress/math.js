pi = Math.PI;

console.log(pi);
console.log(Math.E);
console.log(Math.LN10);
console.log(Math.LN2);
console.log(Math.SQRT2);

console.log(Math.cos(pi));
console.log(Math.sin(pi));
console.log(Math.tan(pi));

console.log(Math.abs(-5.4));

console.log(Math.ceil (5.4));
console.log(Math.floor(5.4));
console.log(Math.round(5.4));
