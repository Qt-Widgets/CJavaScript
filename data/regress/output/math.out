3.14159
2.71828
2.30259
0.693147
1.41421
-1
1.22465e-16
-1.22465e-16
5.4
6
5
5
