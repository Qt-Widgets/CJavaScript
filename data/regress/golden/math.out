3.141592653589793
2.718281828459045
2.302585092994046
0.6931471805599453
1.4142135623730951
-1
1.2246063538223773e-16
-1.2246063538223773e-16
5.4
6
5
5
